fdbbackup/FileDecoder.actor.cpp

```diff
@@ -161,7 +168,13 @@ std::vector<MutationRef> decode_value(const StringRef& value) {
 
 	reader.consume<uint64_t>(); // Consume the includeVersion
 	uint32_t val_length = reader.consume<uint32_t>();
-	ASSERT(val_length == value.size() - sizeof(uint64_t) - sizeof(uint32_t));
+	if (val_length != value.size() - sizeof(uint64_t) - sizeof(uint32_t)) {
+		TraceEvent("ValueError")
```

Comment:
shouldn't this be SevError?

Reply:
Yes. I have an ASSERT below does the same. I'll convert to SevError.